stingray/io.py

```diff
@@ -381,10 +479,10 @@ def lcurve_from_fits(
 def load_events_and_gtis(
     fits_file,
     additional_columns=None,
-    gtistring="GTI,GTI0,STDGTI",
+    gtistring=None,
     gti_file=None,
-    hduname="EVENTS",
-    column="TIME",
+    hduname=None,
+    column=None,
```

Comment:
I added some words in the docs. Nothing to be worried about. The HDU is by default what the mission database says, or the first extension if not in the mission database. For the column, it's the same (mission database, otherwise TIME)